src/tile-link/_tile-link-with-background.scss

```diff
@@ -53,5 +54,25 @@
       opacity: 1;
     }
   }
+
+  .tile-legal {
+    bottom: 1vw;
+    left: 1.5vw;
+    position: absolute;
+    color: $dark-grey;
+    font-size: 13px;
+    @include font($skytext-reg);
```

Comment:
Is there any reason the <code>@include</code> is above other styles? Usually try and keep includes as the last thing after the styles... Doesn't really matter though. Suppose it's just opinion.